open-reac/src/test/java/com/powsybl/openreac/parameters/input/VoltageLevelLimitsOverrideInputTest.java

```diff
@@ -190,6 +190,44 @@ void testVoltageOverrideWithLowLimitGreaterHighLimit() {
         assertEquals("Override on voltage level " + vl.getId() + " leads to low voltage limit >= high voltage limit.", e3.getMessage());
     }
 
+    @Test
+    void testVoltageOverrideWithLowLimitGreaterNominalVoltage() {
+        Network network = IeeeCdfNetworkFactory.create57();
+        setDefaultVoltageLimits(network); // set default voltage limits to every voltage levels of the network
+
+        VoltageLevel vl = network.getVoltageLevels().iterator().next();
+        vl.setHighVoltageLimit(480);
+        vl.setLowVoltageLimit(400);
+        vl.setNominalV(350);
+
+        List<VoltageLimitOverride> voltageLimitsOverride = new ArrayList<>();
+        voltageLimitsOverride.add(new VoltageLimitOverride(vl.getId(), VoltageLimitOverride.VoltageLimitType.LOW_VOLTAGE_LIMIT, false, 370.));
+        voltageLimitsOverride.add(new VoltageLimitOverride(vl.getId(), VoltageLimitOverride.VoltageLimitType.HIGH_VOLTAGE_LIMIT, false, 390.));
+
+        // if after override, low limit > nominal voltage, wrong parameters
+        InvalidParametersException e = assertThrows(InvalidParametersException.class, () -> new VoltageLevelLimitsOverrideInput(voltageLimitsOverride, network));
+        assertEquals("Override on voltage level " + vl.getId() + " leads to low voltage limit > nominal voltage.", e.getMessage());
+    }
+
+    @Test
+    void testVoltageOverrideWithHighLimitLessNominalVoltage() {
+        Network network = IeeeCdfNetworkFactory.create57();
+        setDefaultVoltageLimits(network); // set default voltage limits to every voltage levels of the network
+
+        VoltageLevel vl = network.getVoltageLevels().iterator().next();
+        vl.setHighVoltageLimit(480);
+        vl.setLowVoltageLimit(400);
```

Comment:
Same comment as above, it is weird to start with an inconsistent situation.

Reply:
Changed